### PR TITLE
Adds the yamlitos for WI DPH

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR adds the two WI transports in both staging and prod 

## Changes
- Adds yamlito for WI in prod
- Adds yamlito for WI in staging
- Syncs org files

### Testing
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Security
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### Process
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Security
*List potential security threats and mitigations in the PR*

## Fixes
*List GitHub issues this PR fixes*
- #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: prime-router/settings/prod/0017-wi-receivers.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        operation: "MERGE"
+        numberPerDay: 12
+        initialTime: "01:15"
+        timeZone: "EASTERN"
+        maxReportCount: 100
+      description: ""
+      transport:
+        host: "slhsftp.slh.wisc.edu"
+        port: "2022"
+        filePath: "./Wisconsin_Prime/prod/incoming"
```
Comment (jimduff-usds) [inline prime-router/settings/prod/0017-wi-receivers.yml:57]: Is this right?    I would have expected a ..../test/.... in there.

File: prime-router/settings/organizations-local.yml [hunk 2]
```diff
@@ -544,9 +544,37 @@
         topic: covid-19
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-local.yml:544]: I plan to merge organizations.yml and organizations-local.yml into one file, when I get a chance - I don't think there's a need to keep both.

File: prime-router/settings/organizations-test.yml [hunk 3]
```diff
@@ -495,6 +495,1112 @@
         initialTime: 00:00
         timeZone: EASTERN
 
+
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-test.yml:498]: I think we should delete this file, now that settings table is source of truth.

**Review by jimduff-usds — CHANGES_REQUESTED** [review]
Can you check the directory in prod/0017-wi-receivers.yml, per my comment?  Thanks

File: prime-router/settings/organizations-local.yml (hunk 2 above)
Comment (MauriceReeves-usds) [reply prime-router/settings/organizations-local.yml:544]: I agree. We need to sync them all up and clean them up.

**Review by jimduff-usds — APPROVED** [review]

**Pull request merged**